vocab/security/vocabulary.yml

```diff
@@ -215,51 +252,171 @@ class:
+
+  - id: proof
+    label: Proof sets
+    range: sec:ProofGraph
+    comment: The value of the `proof` property MUST identify <a href=”#ProofGraph”>`ProofGraph` instances</a> (informally, it indirectly identifies <a href=”#Proof”>`Proof` instances</a> each contained in a separate graph). The property is used to associate a proof with a graph of information. The proof property is typically not included in the canonicalized graphs that are then digested, and digitally signed. The order of the proofs is not relevant.
+
+  - id: proofChain
+    label: Proof chain
+    range: rdf:List
+    comment: |
+      The value of the `proofChain` property MUST identify a sequence of <a href=”#ProofGraph”>`ProofGraph` instances</a> (informally, it indirectly identifies a series of <a href=”#Proof”>`Proof` instances</a>, each contained in a separate graph). The property is used to provide a sequence of multiple proofs where the order of when the proofs occurred matters. The proof chain property is typically not included in the canonicalized graphs that are then separately digested, and digitally signed.
+
+  - id: controller
+    label: Controller
+    domain: sec:VerificationMethod
+    range: IRI
+    comment: |
+      A controller is an entity that claims control over a particular resource. Note that control is best validated as a two-way relationship where the controller claims control over a particular resource, and the resource clearly identifies its controller.
+
+  - id: authentication
+    label: Authentication method
+    range: VerificationMethod
+    comment: An `authentication` property is used to specify a URL that contains information about a `verificationMethod` used for authentication.
+
+  - id: assertionMethod
+    label: Assertion method
+    range: VerificationMethod
+    comment: An `assertionMethod` property is used to specify a URL that contains information about a `verificationMethod` used for assertions.
+
+  - id: capabilityDelegation
+    label: Capability Delegation Method
+    range: VerificationMethod
+    comment: |
+      A `capabilityDelegation` property is used to express that one or more `verificationMethods` are authorized to verify cryptographic proofs that were created for the purpose of delegating capabilities.
+      A `verificationMethod` may be referenced by its identifier (a URL) or expressed in full.
+      The aforementioned proofs are created to prove that some entity is delegating the authority to take some action to another entity. A verifier of the proof should expect the proof to express a `proofPurpose` of `capabilityDelegation` and reference a `verificationMethod` to verify it. The dereferenced `verificationMethod` MUST have a controller property that has a property of `capabilityDelegation` that references the `verificationMethod`. This indicates that the controller has authorized it for the expressed `proofPurpose`.
```

Comment:
These are clearly meant to be distinct paragraphs, but that is not communicated by single line-breaks. I am not fluent in YML, and do not know how to embed blank lines in a value, which would *probably* be the easiest solution. (Such blank lines are often taller or shorter than `<p>` spacing in a given CSS, but they do make the paragraph break clear.)

Reply:
@TallTed I am not sure what you refer to, but I _think_ you have not looked at the generated HTML file of the vocabulary. The [preview of the section](https://raw.githack.com/w3c/vc-data-integrity/vocab/sync-with-fpwd/vocab/security/vocabulary.html#capabilityInvocation) clearly separates the distinct paragraphs imho...

Could you check your formatting comments against the generated HTML? Thx.

Reply:
Ah, you're right!  My bad.